Tests/UITests/TestApp/Views/OnboardingWelcomeTestView.swift

```diff
@@ -36,14 +36,3 @@ struct OnboardingWelcomeTestView: View {
             #endif
     }
 }
-
-
-#if DEBUG
-#Preview {
-    OnboardingStack(startAtStep: OnboardingWelcomeTestView.self) {
```

Comment:
We don't offer any alternative for this API, right? Should we keep these previews with just the respective single view in there?

Reply:
if this were in the package itself, i'd agree with you, but since this is the TestApp, which is itself already a playground for testing out the APIs, i'd argue that it's fine not to have the previews at all.

Reply:
I would vote to keep them, but i can understand your rational. Up to you